rs/node_rewards/canister/api/src/provider_rewards_calculation.rs

```diff
@@ -0,0 +1,16 @@
+use crate::DayUtc;
+use candid::{CandidType, Deserialize, Principal};
+use ic_node_rewards_canister_protobuf::pb::rewards_calculator::v1::NodeProviderRewards;
+
+#[derive(CandidType, Clone, Deserialize)]
+pub struct GetNodeProviderRewardsCalculationRequest {
+    pub from: DayUtc,
+    pub to: DayUtc,
```

Comment:
```suggestion
    pub from_nanos: u64,
    pub to_nanos: u64,
```
How about the above? That way the unit is explicit in the field name and we use a simple type for holding the value (it doesn't sound like we'd ever need to extend the record containing the timestamp with anything else).

Reply:
I think it makes sense. Changed it